### ISSUE TITLE
Add the possibility to disable module inspection through a cli flag

### ISSUE THREAD

**Issue by eviln1** [terraform-linters/tflint#1604]
## Introduction

We are using `tflint` both in `pre-commit` hooks and CI pipelines.
Our  `.tflint.hcl` config enables module inspection; this requires `terraform init`, which in turn requires valid credentials to access the configuration's backend.

Most of the developers in our org do not have the required credentials, and their `pre-commit` hooks fail.

We wish to be able to override the  local tflint configuration in a way which isn't currently implemented.

## Proposal

It would be useful to be able to set a `--modules=false` flag, which would take precedence over the `.hcl` config files.

**Comment by wata727** [comment]
Yeah, `--module=false` might be good if we can keep CLI compatibility. But it's probably easier to add `--no-module`. This should be considered by implementation complexity and CLI consistency.

BTW, you can also use `terraform get` instead of `terraform init` for module inspection. The `terraform get` does not initialize the backend and can be run without credentials.
